GeoIP custom endpoint cacert, Update geoip.asciidoc (#105880)
elastic/elasticsearch: docs/reference/ingest/processors/geoip.asciidoc

Note that {es} will make a GET request to `${ingest.geoip.downloader.endpoint}?elastic_geoip_service_tos=agree`,
expecting the list of metadata about databases typically found in `overview.json`.

> The GeoIP downloader uses the JDKs builtin cacerts. If custom endpoint is used add the custom https endpoint cacert(s) to the JDK's truststore.

Reviewer: ```suggestion
The GeoIP downloader uses the JDK's builtin cacerts. If you're using a custom endpoint, add the custom https endpoint cacert(s) to the JDK's truststore.
```

Reviewer: wording nit 🤓
